src/LibVLCSharp.Uno/LibVLCSharp.Uno.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Title>LibVLCSharp.Uno</Title>
     <Summary>Uno integration for LibVLCSharp</Summary>
-    <TargetFrameworks>Xamarin.iOS10;MonoAndroid81</TargetFrameworks>
+    <TargetFrameworks>Xamarin.iOS10;MonoAndroid10.0</TargetFrameworks>
```

Comment:
Uno 3 requires `MonoAndroid10.0` minimum?

Reply:
OK well as not an android developer I didn't fully understand what their requirement was when they moved to android 29 as the min SDK version.   Yes MonoAndroid10/API 29 is the minimum for uno, however the "minSdkVersion" for android is Android 5 or  API 21 (per: https://platform.uno/docs/articles/getting-started/requirements.html).   

Revisiting the earlier discussion if the target sdk does not effect the device minimum do we want to upgrade all projects to 10/11 as only effecting what sdk the user compiling has?

Interestingly the minSdkVersion is not possible to set on any android library only on the final project.  so we only set minSdkVersion on 
`samples/Uno/LibVLCSharp.Uno.Sample.Droid`
`samples/Uno/Sample.MediaPlayerElement/Sample.MediaPlayerElement.Droid`

I will change these to 21, but really any user won't actually know the minSdkVersion.

Google actually requires any app updates or new app submissions to target v29 (again no minimum though).   Starting in august that is v30 for new apps and november v30 for existing.   Uno does support MonoAndroid11 right now but you could wait a few months to make the sdk change there.

It is funny without the ability to specify a minSdkVersion on a lib the app will just crash if it is run on too low of a device when that api is used.

Reply:
> Revisiting the earlier discussion if the target sdk does not effect the device minimum do we want to upgrade all projects to 10/11 as only effecting what sdk the user compiling has?

Samples I don't mind. LibVLCSharp.Uno can target MonoAndroid10. But older non-Uno apps may still use MonoAndroid81 so we should keep it on LibVLCSharp.csproj IMHO.